### ISSUE TITLE
[Feature] rpm kmod build support

### ISSUE THREAD

**Issue by Hero9909** [gnif/LookingGlass#1002]
## Usecase
use kvmfr in fedora silverblue as dkms isn't currently possible  [#1091](https://github.com/coreos/rpm-ostree/issues/1091)
(minor) easier install on rpm based systems

## Possible Solutions
provide a kvmfr.spec for manual rpm-build
or further automatic build via copr 

## Required work
for a manual build, as mentioned above, the kvmfr.spec is required. this provides basic instructions on what files are needed for this kernel module and which have to be installed.

For an automatic build, which of course has to be maintained by someone, an srpm is required. 
This could be also generated by a GitHub action.

## What I have done or what I could already provide
I have already done a manual rpm build, but I'm not sure if everything is done well enough to be used in production.
I can provide a kvmfr.spec, if desired. I already got a build up and running for some frames, but the bleeding edge contains an error about Nvidia's DMA update, which didn't happen before. 

ill test around with an build from the B5 - release version and give an update in the comments

**Comment by Hero9909** [comment]
After some testing, I thought I got the difference that should be the issue: my secondary VM is a QXL-powered one.
But I've tested this behavior with another configuration, I've passed my secondary GPU (gtx950) to this VM, and it's the same issues again. so this is definitely an issue of the Linux build as my connection to the windows VM remains stable.
even though I managed to pass a few frames around, this connection didn't last long.

Currently, I can't disguise if it's a fault in my build / configuration / setup or a deeper error.

<details>
For information, my current setup consists of my host
- AMD 1950X
- GTX 970
- GTX 950

and my configuration for my VMS
Private
- GTX 970
Secondary
- GTX 950 or GTX 970 or QXL
Windows
- GTX 950

If I can help anyone by testing around with a special setup & looking-glass, just ask, ill try.
</details>

Anyway, for a general decision, these issues should not matter and should be resolved in a different thread.
